CHANGELOG-DCA.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release Notes
+=============
+
+.. _Release Notes_1.0.0:
+
+1.0.0
+=====
+
+.. _Release Notes_1.0.0_Prelude:
+
+Prelude
+-------
+
+Released on: 2018-10-18
+
+The Datadog Cluster Agent is compatible with the Datadog Agent 6.5.0.
```

Comment:
Forward-compat is a given for us, but being explicit is probably better. WDYT of something along the lines of `compatible with versions 6.5.1 and up of the Datadog Agent`